src/Elcodi/Component/Menu/Entity/Menu/Node.php

```diff
@@ -290,6 +297,10 @@ public function isActive($currentUrl)
      */
     public function isExpanded($currentUrl)
     {
+        if (!($this->subnodes instanceof Collection)) {
```

Comment:
You could also remove the internal parentheses: `if (!$this->subnodes instanceof Collection) {`